Creating Documentation folder in project root.

Khan-Ramsha opened the pull request:
PR #8 

This PR is dedicated to adding the application documentation to the project.

1) This PR implements the changes mentioned. I have not moved docs/ folder to project root, as docs/ already contains the OPENAPI Documentation code, which is integrated into the app’s source code and actively used by the application.

2) A new documentation/ folder has been created in the project root. This folder contains detailed documentation for the app.

I would appreciate it if you could review the changes and let me know if any modifications are needed. If any changes or adjustments are required, I will address them promptly. Thankyou!

codecakes reviewed (changes requested):
change documentation to docs/

the document should be in markdown format.

Khan-Ramsha commented:
> change documentation to docs/
> 
> the document should be in markdown format.

I’ve updated the documentation to 'docs/' and converted it to markdown format as per your advice. Please review the changes when you have a moment.

codecakes reviewed (approved):
looks good! It would be good to link the docs from README as well.
